go.mod

```diff
@@ -15,43 +15,85 @@ require (
 	github.com/kubernetes-csi/external-snapshotter/client/v4 v4.1.0
 	github.com/prometheus/client_golang v1.11.0
 	github.com/prometheus/client_model v0.2.0
-	github.com/prometheus/common v0.26.0
-	github.com/spf13/cobra v1.1.3
-	golang.org/x/oauth2 v0.0.0-20201208152858-08078c50e5b5 // indirect
+	github.com/prometheus/common v0.28.0
+	github.com/spf13/cobra v1.2.1
+	golang.org/x/oauth2 v0.0.0-20210819190943-2bc19b11175f // indirect
 	google.golang.org/appengine v1.6.7 // indirect
-	google.golang.org/grpc v1.38.0
-	k8s.io/api v0.22.0
-	k8s.io/apimachinery v0.22.0
-	k8s.io/client-go v0.22.0
-	k8s.io/component-base v0.22.0
-	k8s.io/klog/v2 v2.9.0
+	google.golang.org/grpc v1.40.0
+	k8s.io/api v0.23.0
+	k8s.io/apimachinery v0.23.0
+	k8s.io/client-go v0.23.0
+	k8s.io/component-base v0.23.0
+	k8s.io/klog/v2 v2.30.0
 	k8s.io/kubernetes v1.21.0
```

Comment:
Why is this still 1.21?

Reply:
it was a miss, updated this too. @xing-yang